### PR TITLE
Create FromN_2

### PR DESCRIPTION
**Pull request by nmikhailova**
number 3816547290
run time without threads 135661

number 3816547290
run time of tread 97550
run time of tread 97790
run time of tread 97801
run time of tread 98245
run time of tread 98459

### CODE REVIEW

File: nastia/FromN_2 [hunk 1]
```diff
@@ -0,0 +1,102 @@
+package com.company;
+
+/**
+ * Created by NastyaMikhailova on 04.04.2016.
+ */
+public class  MyThread implements Runnable {
```
Comment (pasby) [inline nastia/FromN_2]: classes should be in single files

File: nastia/FromN_2 [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    private long firstNumber, lastNumber, startTime;
+    public MyThread(long firstNumber, long lastNumber) {
+        this.firstNumber = firstNumber;
+        this.lastNumber = lastNumber;
+        this.startTime = System.currentTimeMillis();
+    }
+
+    public synchronized void spanOfThread(long firstNumber, long lastNumber){
+        Challenge element = new Challenge();
+        //System.out.println("fn " + firstNumber + " ln " + lastNumber);
```
Comment (pasby) [inline nastia/FromN_2]: no need unused comments

File: nastia/FromN_2 [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        for (long i = firstNumber; i <= lastNumber; i += 10 ){
+            if (element.check1(i)){
+                if (element.check2(i)){
```
Comment (pasby) [inline nastia/FromN_2]: reuse the same parts of code

**Comment by pasby** [comment]
Please, make classes in different files and in package src/main/java/...

**Comment by nmikhailova** [comment]
I can't to add new files here

**Pull request closed without merging**